Automattic/wp-calypso
# Trail Map: Reinstate "ad-free experience" in the features grid.

Related to #89682

## Proposed Changes

This PR reinstates "ad-free experience" in the features grid of Project Trail Map. More detail can be found in pau2Xa-5Om-p2#comment-15392

For the copy-only variation, the ad-free feature is added to the Starter plan:

![CleanShot 2024-05-06 at 17 13 21@2x](https://github.com/Automattic/wp-calypso/assets/1842898/4f1d92a2-2929-4dbc-8fe8-cc6d5b928292)


For the copy-and-structure variation, it is added to all the paid plans:

![CleanShot 2024-05-06 at 17 14 04@2x](https://github.com/Automattic/wp-calypso/assets/1842898/613cfa22-a9ee-4e70-9999-1f832f4f2a06)


## Testing Instructions


Initiate the plans-grid-next Storybook by running `yarn workspace @automattic/plans-grid-next storybook`

1. Go to the story "Trail Map Copy", and confirm that "ad-free experience" is presented in the Starter plan.
2. Go to the story "Trail Map Copy and Structure", and confirm that "ad-free experience" is presented in all the paid plans.


## Pre-merge Checklist
